lib/file-reader.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const fs = require('fs');
+const index = require('../index.js');
+
+module.exports = exports = {};
+
+exports.readFile = function(file, callback) {
+  fs.readFile(file, function(err, data) {
+    if(err) return callback(err);
+    return callback(null, data.toString('hex', 0, 8));
+  });
+};
```

Comment:
Minor nitpick- you could use module.exports = function( ) here since you are exporting a single method.